shelley/chain-and-ledger/dependencies/non-integer/app/Main.hs

```diff
@@ -116,33 +126,33 @@ main = do
   --       | (i, c, y) <- precomputed
   --       ]
   --   ]
-  precomputedLeader <-
-    forM leaderTest $ \(sigma, p) -> do
-      let q = 1 - p
-      let c = 1 - f
-      let c' = ln' c
-      return (1/q, (-sigma * c'))
-  precomputedLeader' <-
-    forM leaderTest $ \(sigma, p) -> do
-      c <- evaluate $ ln' (1 - f)
-      return (p, sigma, c)
-
-  defaultMain
-    [
-      bgroup
-      "taylorCmp"
-      [bench "cmp" $ whnf leader' (invQ, negAlpha)
-      | (invQ, negAlpha) <- precomputedLeader
-      ]
-    , bgroup
-      "partial pre-computation"
-      [bench "exp(alpha*c)" $ whnf (\(p', s', c') -> p' < 1 - exp' (s' * c')) (p, sigma, c)
-      | (p, sigma, c) <- precomputedLeader'
-      ]
-    , bgroup
-      "full calculation"
-      [bench "cmp" $ whnf (\(s, p') -> p' < 1 - ((1 - f) *** s)) (sigma, p)
-      | (sigma, p) <- leaderTest
-      ]
-    ]
-  print $ map leaderComputation leaderTest
+  -- precomputedLeader <-
```

Comment:
any reason to keep all these comments in?

Reply:
thanks for catching this, I removed all those comments and also unused definitions